source/extensions/tracers/zipkin/zipkin_tracer_impl.h

```diff
@@ -137,10 +137,23 @@ class Driver : public Tracing::Driver {
   TimeSource& time_source_;
 };
 
+/**
+ * Information about the Zipkin collector.
+ */
+struct CollectorInfo {
+  // The Zipkin collector endpoint/path to receive the collected metrics. e.g. /api/v1/spans.
```

Comment:
Not sure the style but this is the intent

```suggestion
  // The Zipkin collector endpoint/path to receive the collected trace data. e.g. /api/v1/spans if HTTP_JSON_V1 or /api/v2/spans otherwise
```